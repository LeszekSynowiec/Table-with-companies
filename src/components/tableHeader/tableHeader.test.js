import React from "react";
import TableHead from "./tableHead.component";
import { shallow } from "enzyme";

describe("TableHead component", () => {
  it("shoud render with out crashing", () => {
    const headersTest = [
      {
        id: "test Id 1",
        name: "test Name 1",
        position: "test Position 1",
      },
      {
        id: "test Id 2",
        name: "test Name 2",
        position: "test Position 2",
      },
      {
        id: "test Id 3",
        name: "test Name 3",
        position: "test Position 3",
      },
    ];
    const component = shallow(<TableHead headers={headersTest} />);
    expect(component).toMatchSnapshot();
  });
});
