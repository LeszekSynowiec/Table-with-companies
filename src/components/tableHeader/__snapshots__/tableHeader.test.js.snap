// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead component shoud render with out crashing 1`] = `ShallowWrapper {}`;
